source/locale/pt_BR/LC_MESSAGES/flow.po

```diff
@@ -1221,14 +1336,14 @@ msgid ""
 "generated, as long as the accumulated value is greater than 100. See "
 "image :numref:`using_cum_sum_result`."
 msgstr ""
-"Portanto, se a estação meteorológica (dispositivo definido no nó do dispositivo de evento com"
-"publicação marcada) envia várias mensagens como  `{\"rain\": 5}` durante"
-"a última hora e em uma dessas vezes a soma for superior a 100, o"
-"uma notificação será gerada. Nota: Várias notificações podem ser"
-"gerado, desde que o valor acumulado seja superior a 100 na última hora. Veja"
-"image :numref:`using_cum_sum_result`. "
+"Portanto, se a estação meteorológica (dispositivo definido no nó do "
+"dispositivo de evento compublicação marcada) envia várias mensagens como"
```

Comment:
> dispositivo de evento **com publicação** marcada...

Reply:
missing a space at the end of the line before closing the quotes...

The space is at the beginning of the bottom line, but it is better to follow the pattern of leaving at the end of the line, what do you think?